src/css/styles.scss

```diff
@@ -0,0 +1,180 @@
+*,
+*:before,
```

Comment:
This is hard to do without a visual - maybe we can look at what it looks like in the morning before we officially merge!